Code/Engine/Core/World/Implementation/GameObject.cpp

```diff
@@ -145,9 +171,29 @@ void ezGameObject::Reflection_SetMode(ezObjectMode::Enum mode)
   }
 }
 
-ezGameObject* ezGameObject::Reflection_FindChildByName(ezStringView sName, bool bRecursive)
+ezGameObject* ezGameObject::Reflection_GetParent() const
+{
+  return GetWorld()->GetObjectUnchecked(m_uiParentIndex);
+}
+
+void ezGameObject::Reflection_SetGlobalPosition(const ezVec3& vPosition)
+{
+  SetGlobalPosition(vPosition);
+}
+
+void ezGameObject::Reflection_SetGlobalRotation(const ezQuat qRotation)
+{
+  SetGlobalRotation(qRotation);
+}
+
+void ezGameObject::Reflection_SetGlobalScaling(const ezVec3 vScaling)
+{
+  SetGlobalScaling(vScaling);
+}
```

Comment:
Because we have two overloads of SetGlobal... one with regular math classes, one with simd math classes. This doesn't work with the reflection since it doesn't know which one to pick. The missing & is wrong of course.